### ISSUE TITLE
[Feature]: SHIFT or CTRL modifier for click event/onValueChange()

### ISSUE THREAD

**Issue by brightloudnoise** [tremorlabs/tremor-npm#842]
### What problem does this feature solve?

Adding the click event/onValueChange functionality in #195 gives us some great opportunities for interaction. I've been playing with it in the recent beta.

Have you also considered returning the click event so we can detect if the CTRL or SHIFT modifier key is pressed? This would let us select more than one element in a chart to expose those selections as a filter.

**Comment by mgguo** [comment]
Agreed. Having the raw click event wrapped in the payload would support many interaction scenarios, e.g. open a context menu from where it's clicked
